f/frizzle/kobo_responses.script.yaml

```diff
@@ -34,18 +33,14 @@ schema:
       description: ''
       default: null
       originalType: string
-    kobo_api_key:
-      type: string
+    kobo_server:
```

Comment:
I think `kobo_server` could be confusing for cases where a user has multiple accounts (and therefore API keys) on the same server.

I'm thinking of something like `kobo_configuration`. Perhaps with the description of what that actually entails -- e.g. a server/key pairing -- that could be sufficient?

Reply:
`kobo_account`?  As I look at the Kobo server UI, there is exactly 1 API key per user account.

The other possibility is just `kobo`.  Most of the resource types in the Windmill hub do not specify beyond the service name, even though they actually represent one account or one API Key on that service.

I suggest we go with `kobo_account` because it's less ambiguous, but happy to change.

Reply:
I'm good with `kobo_account` and a description that describes that this resource includes both a URL and an API key.